docker-compose.airflow.yml

```diff
@@ -0,0 +1,155 @@
+version: '2'
+services:
+  redis:
+    image: redis:3-alpine
+
+  nginx: 
```

Comment:
This needs to exist to override the `extra_hosts` provided in `docker-compose.yml`, and to add links to the `airflow` services (which no longer exist in the main `docker-compose` file).